tools/devops/automation/templates/mac/build.yml

```diff
@@ -128,6 +128,12 @@ steps:
   timeoutInMinutes: 30
   enabled: false
 
+- bash: |
+    sudo rm -Rf $(Build.SourcesDirectory)/package
```

Comment:
question: does the script get run as root, and if so, why does it need `sudo` and if not, how does it get a password?

Reply:
It is indeed ran as sudo and is not needed AFAIK, but it does say paranoid in the title hehe